### PR TITLE
Help updates for SAM 2023.12.17 r1

### PR DESCRIPTION
**Pull request by cpaulgilman**
Clean up window reference topics.

Detailed PV time series results, also note lat/long/elev/tz pass through from weather file

Fix PV land area and GCR descriptions.

Revise Solar Resource topic

Electric Load subhourly weather file available via solar resource Advanced Donwload option.

### CODE REVIEW

File: deploy/runtime/help/html/pv_system_design.html [hunk 1]
```diff
@@ -691,30 +692,34 @@ <h4 class="p_Heading4"><span class="f_Heading4">Electrical Sizing Information an
...
+<p class="p_VariableDescription"><span class="f_VariableDescription">For PVWatts, the AC capacity is the system nameplate capacity divided by the DC to AC ratio.</span></p>
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Land area per system capacity</span></p>
-<p class="p_VariableDescription"><span class="f_VariableDescription">When you choose </span><span class="f_CHInterfaceElement">Enter area per capacity in acres/MWe</span><span class="f_VariableDescription">, type the land use estimate in acres per AC megawatt of system capacity.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">When you choose </span><span class="f_CHInterfaceElement">Enter area per capacity in acres/MWac</span><span class="f_VariableDescription">, type the land use estimate in acres per AC megawatt of system capacity. Typical values range between 5 and 10 acres/MWac.</span></p>
```
Comment (brtietz) [inline deploy/runtime/help/html/pv_system_design.html:711]: I might recommend knocking the lower limit down to 4. The latest report from the reV team has 4.3 acres/MWac (assuming an ILR of 1.34) https://www.nrel.gov/docs/fy24osti/87843.pdf

File: deploy/runtime/help/html/pvwatts_system_design.html [hunk 2]
```diff
@@ -669,30 +669,34 @@ <h3 class="p_Heading3_atocs_" style="page-break-inside: avoid; page-break-after:
...
+<p class="p_VariableDescription"><span class="f_VariableDescription">For PVWatts, the AC capacity is the system nameplate capacity divided by the DC to AC ratio.</span></p>
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Land area per system capacity</span></p>
-<p class="p_VariableDescription"><span class="f_VariableDescription">When you choose </span><span class="f_CHInterfaceElement">Enter area per capacity in acres/MWe</span><span class="f_VariableDescription">, type the land use estimate in acres per AC megawatt of system capacity.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">When you choose </span><span class="f_CHInterfaceElement">Enter area per capacity in acres/MWac</span><span class="f_VariableDescription">, type the land use estimate in acres per AC megawatt of system capacity. Typical values range between 5 and 10 acres/MWac.</span></p>
```
Comment (brtietz) [inline deploy/runtime/help/html/pvwatts_system_design.html:688]: If we change detailed PV based on my comment above, this also needs to change.

**Review by brtietz — APPROVED** [review]
Thanks for updating! I have a few nitpicks and found one typo, but still happy to approve.

**Review by sjanzou — APPROVED** [review]
Looks good for patch 1 and release candidate.
A few comments from Brian that we can update after testing release candidate.

**Pull request merged**